src/server/game/Spells/Auras/SpellAuraEffects.cpp

```diff
@@ -869,7 +869,8 @@ void AuraEffect::PeriodicTick(AuraApplication* aurApp, Unit* caster) const
     Unit* target = aurApp->GetTarget();
 
     // Update serverside orientation of tracking channeled auras on periodic update ticks
-    if (caster && m_spellInfo->IsChanneled() && m_spellInfo->HasAttribute(SPELL_ATTR1_CHANNEL_TRACK_TARGET))
+    // exclude players because can turn during channeling and shouldn't desync orientation client/server
+    if (caster && caster->GetTypeId() != TYPEID_PLAYER && m_spellInfo->IsChanneled() && m_spellInfo->HasAttribute(SPELL_ATTR1_CHANNEL_TRACK_TARGET))
```

Comment:
!caster->IsPlayer()   we're not in 2008 anymore

Reply:
thx ;)